cristin-import/src/test/java/no/unit/nva/cristin/lambda/FilenameEventEmitterTest.java

```diff
@@ -73,15 +77,18 @@ public void handlerThrowsNoExceptionWhenInputIsValid() {
         assertDoesNotThrow(action);
     }
 
-    @Test
-    public void handlerEmitsEvensForEveryFilenameInS3BucketWhenInputIsAnExistingNotEmptyS3Location()
+    @ParameterizedTest(
+        name = "handlerEmitsEventsWithFullFileUriForEveryFilenameInS3BucketWhenInputIsAnExistingNotEmptyS3Location")
+    @ValueSource(strings = {SOME_S3_LOCATION, SOME_S3_LOCATION + PATH_SEPARATOR})
```

Comment:
What you're testing here seems to be that the s3-location is valid and has an optional trailing path separator. Consequently, the name of the test should be changed and the ValueSource should be empty string and path separator